src/components/Banner/ErrorBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { BiErrorCircle } from "react-icons/bi";
+
+type ErrorProps = {
+  children: React.ReactNode;
+};
+
+const StyledBanner = styled.div`
+  display: flex;
+  align-items: center;
+  background-color: #f3a2a2;
+
+  margin: 20px;
+  padding: 10px;
+  border-radius: 15px;
+
+  div {
+    margin: 0px 10px;
+  }
+`;
+
+export default function ErrorBanner({ children }: ErrorProps) {
```

Comment:
```suggestion
type ErrorBannerProps = {
  children: React.ReactNode;
};

const StyledErrorBanner = styled.div`
  display: flex;
  align-items: center;
  background-color: #f3a2a2;

  margin: 20px;
  padding: 10px;
  border-radius: 15px;

  div {
    margin: 0px 10px;
  }
`;

export default function ErrorBanner({ children }: ErrorBannerProps) {
```

just to keep it aligned everywhere

Reply:
✅